### DISCUSSION TITLE
Should /.nx-container be checked-in

### DISCUSSION THREAD

**Discussion by vicb** [gperdomor/oss#706] [Q&A] [answered]
I am using your great plugin to create Docker images.

I have noticed that it creates a `.nx-container` folder at the root of the project.

I don't think this needs to be checked-in, right ?
If so what about moving it to `/node_modules/.cache/nx-tools` ?

**Comment by gperdomor** [comment, marked as answer]
Sounds good to me...

https://github.com/gperdomor/nx-tools/pull/708
